cmd/reporter/root.go

```diff
@@ -50,11 +50,12 @@ type GlobalOpts struct {
 func newRootCmd(out io.Writer, args []string) (*cobra.Command, error) {
 	global = new(GlobalOpts)
 	cmd := &cobra.Command{
-		Use:              "merkely",
-		Short:            "The Merkely evidence reporting CLI.",
-		Long:             globalUsage,
-		SilenceUsage:     true,
-		TraverseChildren: true,
+		Use:               "merkely",
+		Short:             "The Merkely evidence reporting CLI.",
+		Long:              globalUsage,
+		SilenceUsage:      true,
+		TraverseChildren:  true,
+		DisableAutoGenTag: true,
```

Comment:
this is the only line you need to disable the autogen tag. Since this is the root command, all others will respect this without explicitly setting it on each of them.